src/oncoexporter/cda/cda_mutation_factory.py

```diff
@@ -11,44 +12,40 @@ class CdaMutationFactory(CdaFactory):
     https://cda.readthedocs.io/en/latest/Schema/fields_mutation/
     153 fields - need to decide which to keep
 
-    cda_subject_id
-    primary_site
-    Hugo_Symbol
-    Entrez_Gene_Id
-    NCBI_Build
-    Chromosome
-    Start_Position
-    End_Position
-    Reference_Allele
-    Tumor_Seq_Allele1
-    Tumor_Seq_Allele2
-    dbSNP_RS
-    dbSNP_Val_Status
-    Match_Norm_Seq_Allele1
-    Match_Norm_Seq_Allele2
-    Tumor_Validation_Allele1
-    Tumor_Validation_Allele2
-    Match_Norm_Validation_Allele1
-    Match_Norm_Validation_Allele2
-    Mutation_Status
-    HGVSc
-    HGVSp
-    HGVSp_Short
-    Transcript_ID
-    ENSP
-
+    'Entrez_Gene_Id',
+    'Hugo_Symbol',
+    'NCBI_Build',
+    'Chromosome',
+    'Start_Position',
+    'Reference_Allele',
+    'Tumor_Seq_Allele2',
+    'dbSNP_RS',
+    'Transcript_ID',
+    'HGVSc',
+    'ENSP',
+    'HGVSp_Short',
+    'Mutation_Status',
+    't_depth',
+    't_ref_count',
+    't_alt_count',
+    'n_depth',
+    'n_ref_count',
+    'n_alt_count',
     """
 
     def __init__(self):
         self._column_names = [
-            'cda_subject_id', 'primary_site', 'Hugo_Symbol', 'Entrez_Gene_Id', 'NCBI_Build', 'Chromosome',
-            'Start_Position', 'End_Position', 'Reference_Allele', 'Tumor_Seq_Allele1', 'Tumor_Seq_Allele2', 'dbSNP_RS',
-            'dbSNP_Val_Status', 'Match_Norm_Seq_Allele1', 'Match_Norm_Seq_Allele2', 'Tumor_Validation_Allele1',
-            'Tumor_Validation_Allele2', 'Match_Norm_Validation_Allele1', 'Match_Norm_Validation_Allele2',
-            'Mutation_Status', 'HGVSc', 'HGVSp', 'HGVSp_Short', 'Transcript_ID', 'ENSP'
+            'Entrez_Gene_Id', 'Hugo_Symbol',
+            'NCBI_Build', 'Chromosome', 'Start_Position', 'Reference_Allele', 'Tumor_Seq_Allele2',
+            'dbSNP_RS',
+            'Transcript_ID', 'HGVSc', 'ENSP', 'HGVSp_Short',
+            'Mutation_Status',
+            't_depth', 't_ref_count', 't_alt_count',
+            'n_depth', 'n_ref_count', 'n_alt_count'
         ]
+        self._logger = logging.getLogger(__name__)
 
-    def to_ga4gh(self, row) -> pp.VariationDescriptor:
+    def to_ga4gh(self, row: pd.Series) -> pp.VariantInterpretation:
         """
         convert a row from the CDA mutation table into an
         Individual message (GA4GH Phenopacket Schema)
```

Comment:
s/Individual/VariantInterpretation/

Reply:
Very, very very minimal & cryptic :smile: 
A puzzle of a day!